.env

```diff
@@ -4,6 +4,9 @@ NOTIFIER_PUBSUB_TOPIC_NAME = notifier-topic
 
 ROUTE_CHECKER_PUBSUB_SUBSCRIPTION_NAME = route-checker-subscription
 ROUTE_CHECKER_PUBSUB_TOPIC_NAME = route-checker-topic
+ROUTE_CHECKER_API_KEY = 5ae2e3f221c38a28845f05b6a2b7dfab4d46dcb10528c55de3475836
```

Comment:
Please, avoid uploading the API KEY to the repo directly.
This should be stored secretly and they can ban us from using the API due to exposing this key. 🙇🏼